src/entities/returnUser.ts

```diff
@@ -0,0 +1,26 @@
+import "reflect-metadata";
+import { Entity, Column } from "typeorm";
+
+@Entity({ name: "returnUser" })
+export class ReturnUser {
+  @Column()
+  id: number;
+
+  @Column({
+    length: 100,
+  })
+  name: string;
+
+  @Column({
+    length: 50,
+  })
+  email: string;
+
+  @Column()
+  age: number;
+
+  @Column({
+    nullable: true,
```

Comment:
alias, seu linter nao ta ajustando? to achando estranho pular essas linhas aqui, poderia ser tudo da linha 22

Reply:
O linter estava editando assim por causa da vírgula, removi e agora tudo está na linha 22